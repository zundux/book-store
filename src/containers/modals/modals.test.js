import React from "react";
import Modals from "./modals";

describe("Modals container", () => {

  it("Modals container", () => {
    const component = shallow(<Modals/>);
    matchSnapshot(component);
  });
});