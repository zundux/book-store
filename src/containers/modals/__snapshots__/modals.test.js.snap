// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modals container Modals container 1`] = `
<div
  style={
    Object {
      "display": "none",
    }
  }
/>
`;
